lib/python/ensembl/brc4/runnable/utils.py

```diff
@@ -0,0 +1,44 @@
+#!env python3
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Any
+from pathlib import Path
+import json
+
+
+class Utils:
```

Comment:
The following 2 methods could be just methods of the module `utils.py` without the need to be part of a class, but this is ok as well.

Reply:
I've changed it to a simpler module